packages/blade/src/utils/makeSpacingSize/makeSpacingSize.web.ts

```diff
@@ -0,0 +1,6 @@
+const makeSpacingSize = (size: number): `${number}rem` => {
```

Comment:
This is exactly the same as `makeTypographySize`, should we club them both together and call it `makeSize`?

Reply:
nupe. The reason is that we use `rems` in typography but in layout we need to use `px`. [Refer RFC](https://github.com/razorpay/blade/blob/master/rfcs/2021-02-19-units-for-typography-and-layout.md#what-will-work-for-us)

Reply:
Also we can just rename this to `makeSpace`. Size is something that will come separately and can be something as `makeSize` and will affect things like `height`, `width`, etc family

Reply:
Oh got it. I'll rename this and return `px`

Reply:
Also, since `makeSpace` would be the same for native & web I'm keeping only 1 implementation